frontend/src/metabase/components/ItemsTable/BaseItemsTable.unit.spec.tsx

```diff
@@ -75,7 +76,7 @@ describe("Collections BaseItemsTable", () => {
 
   it("displays item data", () => {
     setup();
-    const lastEditedAt = moment(timestamp).format("MMMM DD, YYYY");
+    const lastEditedAt = moment(timestamp).format("MMMM D, YYYY");
```

Comment:
Since this branch uses `<DateTime unit="day" value={lastEditInfo.timestamp} />` to display the "last edited at" datetime, the formatting has now been corrected from "April 05, 2024" to "April 5, 2024", hence this change